File: docs/evidence/2026-09-25-rust-core-spikes/swig/php/t.php
<?php
$b = preg_replace('/\s/', '', file_get_contents("../../../../../fixtures/public-receipts/receipt-sandbox-g5.b64"));
echo "php " . PHP_VERSION . ": " . substr(verify_receipt_base64(aprv_verifier_new_receipt("dev.bonzer.weeka.app"), $b), 0, 40) . "\n";
try { verify_receipt_base64(aprv_verifier_new_receipt("x.y"), $b); } catch (Throwable $e) { echo "php error ok: " . get_class($e) . " " . $e->getMessage() . "\n"; }

File: docs/evidence/2026-09-25-rust-core-spikes/neutral-surface/py/t.py
from aprv_uniffi_remote import verify_base64, VerifyError, Reason
b = "".join(open("$REPO/fixtures/public-receipts/receipt-sandbox-g5.b64").read().split())
r = verify_base64("dev.bonzer.weeka.app", b); print("ok:", r.bundle_id, r.creation_date_ms, r.in_app_purchase_count)
try: verify_base64("x.y", b)
except VerifyError.Verification as e: print("error ok:", e.reason)
